### PR TITLE
Allow generators as parameters to setMultiple

### PR DESCRIPTION
**Pull request by asgrim**
More of #8.

### CODE REVIEW

File: test/unit/CacheIntegrationTest.php [hunk 1]
```diff
@@ -25,8 +25,6 @@ protected function setUp() : void
         parent::setUp();
 
         // @todo: Let's make these tests pass
-        $this->skippedTests['testSetMultipleWithGenerator'] = true;
```
Comment (Ocramius) [inline test/unit/CacheIntegrationTest.php]: Don't we also need unit tests in our suite?

Comment (asgrim) [inline test/unit/CacheIntegrationTest.php]: Yes, I'll add

Comment (asgrim) [inline test/unit/CacheIntegrationTest.php]: Added \m/

**Comment by Ocramius** [comment]
👍

**Pull request merged**